src/TServer.cpp

```diff
@@ -365,3 +424,21 @@ void TServer::InsertClient(const std::shared_ptr<TClient>& NewClient) {
     WriteLock Lock(mClientsMutex); // TODO why is there 30+ threads locked here
     (void)mClients.insert(NewClient);
 }
+
+void TServer::HandlePosition(TClient& c, std::string Packet) {
```

Comment:
avoid a copy of the entire packet string (expensive) by passing by (const) reference.
anything that isn't an int/uint/size_t/bool/ptr should be passed by const& unless semantically incorrect
```suggestion
void TServer::HandlePosition(TClient& c, const std::string& Packet) {
```